index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Hiwot's Portfolio</title>
+ 
+<link rel="stylesheet" href="styles.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+</head>
+
+<body> 
+    <header class="navigation">
+        <nav class="toolbar-container">
+            <div class="logo"><a href="#welcome">Welcome</a></div>
+            <div>
+                <a class="icon" ><a href="#menu-bar">
+                <i class="fa fa-bars"></i></a>
+            </div>
+        </nav>
+    </header>
+    <section class="headline back-image">
+        <div class="primary-headline" id="desktop-primary">
+            <h1>Hey there. I'm <br> Hiwot<span class="h1color"><br>
+             I'm a software developer</span>
+            </h1>
+        </div>
+         <div>
+            <p class="supporting-text">  I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div>
+            <ul class="socialt">
+                <li> <a href="#"><img src="image/twitter.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/vector.png" /></a> </li>
+                <li> <a href="#"><img src="image/Vector (1).png" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/medium.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/github.svg" /></a> &nbsp; &nbsp; </li>
+                <li> <a href="#"><img src="image/angellist.svg" /></a> &nbsp; </li>
+            </ul> 
+        </div>
+        <div class="scroll">
+            <img src="image/Disabled.png" alt="Scroll down" >
+        </div>
+    </section>
+</body>
+</html>
```

Comment:
- [ ] [OPTIONAL] Good job so far, your work is impressive but please make sure that your mobile design is minimum mobile screen size should be 375px. :+1: 

#### Figma design
![Screenshot from 2022-10-26 13-18-36](https://user-images.githubusercontent.com/6615394/198038034-e5df6037-501f-4b7e-9a71-0e75be0c32bd.png)

#### Your design
![Screenshot from 2022-10-26 13-06-11](https://user-images.githubusercontent.com/6615394/198038042-bba1210b-3982-49ee-84fa-815687545145.png)